compiler/rustc_mir_transform/src/coverage/counters.rs

```diff
@@ -114,6 +130,80 @@ impl CoverageCounters {
         self.next_expression_id = next.next_id();
         next
     }
+
+    fn set_bcb_counter(
+        &mut self,
+        bcb: BasicCoverageBlock,
+        counter_kind: CoverageKind,
+    ) -> Result<Operand, Error> {
```

Comment:
This method is mostly a direct copy of the old `BasicCoverageBlockData::set_counter`, which was moved here from `graph.rs`.